spec/models/manageiq/providers/openstack/infra_manager/refresher_rhos_juno_spec.rb

```diff
@@ -80,7 +80,7 @@ def assert_table_counts
     expect(VmOrTemplate.count).to                be > 0
     expect(OperatingSystem.count).to             be > 0
     expect(Hardware.count).to                    be > 0
-    expect(Disk.count).to                        be > 0
+    expect(Disk.count).to                        eq 0
```

Comment:
@tzumainn seems like collection of disks is broken? https://github.com/Ladas/manageiq/blob/9fdcaab560de3689dfb5aeeda9a61320c1f62d1f/app/models/manageiq/providers/openstack/infra_manager/refresh_parser.rb#L244-L244